### PR TITLE
BFB-430: Back Corridor and Foodcourt second iteration

### PR DESCRIPTION
**Pull request by csweet22**
**JIRA Ticket**
https://perigongames.atlassian.net/browse/BFB-430

**Description**
As long as nothing is broken, I just need this merged so that Magan can start working on her portion.

NavMesh still hasn't been fully remade and is still somewhat broken.

This is just a secondary pass where the areas have slightly more detail. Still no textures or more final meshes.

### CODE REVIEW

File: Assets/Editor/DevBuildScript.cs [hunk 1]
```diff
@@ -21,8 +21,8 @@ public static void MyBuild()
         }
 
         buildPlayerOptions.scenes = scenePaths.ToArray();
-        //buildPlayerOptions.locationPathName = "build/StandaloneWindows64/windows.exe";
-        buildPlayerOptions.locationPathName = "C:/Users/Developer/OneDrive/_PerigonGames/new_dev_build/windowsBuild.exe";
+        buildPlayerOptions.locationPathName = "build/StandaloneWindows64/windows.exe";
+        // buildPlayerOptions.locationPathName = "C:/Users/Developer/OneDrive/_PerigonGames/new_dev_build/windowsBuild.exe";
```
Comment (m4rtint) [inline Assets/Editor/DevBuildScript.cs:25]: Do you know how to revert this file?

File: Assets/_BForBoss/Tests/CharacterSceneTests/CharacterMovementTests.cs [hunk 2]
```diff
@@ -25,94 +24,73 @@ public override void Setup()
         [UnityTest]
         public IEnumerator Character_MoveForward_GreaterZPosition()
         {
-            for (int i = 0; i < 50; i++)
+            for (int i = 0; i < 5; i++)
```
Comment (m4rtint) [inline Assets/_BForBoss/Tests/CharacterSceneTests/CharacterMovementTests.cs:27]: Also revert this file

**Comment by m4rtint** [comment]
I'll need to do a clean up of ur PR

**Comment by csweet22** [comment]
> Clean Up PR to make sure Develop isn't over written

Oh wow I didn't realize this branch got so messed up! I'll try reverting the merge from develop into this branch to see if maybe that caused it?

File: Assets/_BForBoss/_Character/Scripts/InputSettings.cs [hunk 3]
```diff
@@ -0,0 +1,123 @@
+using ECM2.Components;
+using Perigon.Utility;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Perigon.Character
+{
+    public class InputSettings : IInputSettings
```
Comment (m4rtint) [inline Assets/_BForBoss/_Character/Scripts/InputSettings.cs:8]: Delete this.

**Comment by m4rtint** [comment]
Also have to be careful to update develop before merging develop

**Review by m4rtint — APPROVED** [review]

**Pull request merged**